log/gokit_test.go

```diff
@@ -5,14 +5,18 @@
 package log
 
 import (
+	"bytes"
+	"fmt"
 	"testing"
+	"time"
 
 	"github.com/go-kit/log"
 	"github.com/go-kit/log/level"
+	"github.com/stretchr/testify/require"
 )
 
 func BenchmarkDebugf(b *testing.B) {
```

Comment:
[nit] Rename this.

Reply:
I have renamed the benchmark, and also changed its implementation: I replaced the `fmt.Sprintf` usage with `LazySprintf`.